test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I don't think we need to be so "strict". Actually, the probability of name collision is very low, only in cases like this (a .test that is generated from another one, with the idea of keeping the same name) we need a prefix.